packages/now-static-build/src/index.ts

```diff
@@ -407,10 +415,35 @@ export async function build({
   throw new Error(message);
 }
 
-export async function prepareCache({ workPath }: PrepareCacheOptions) {
-  return {
-    ...(await glob('node_modules/**', workPath)),
-    ...(await glob('package-lock.json', workPath)),
-    ...(await glob('yarn.lock', workPath)),
-  };
+export async function prepareCache({
+  entrypoint,
+  workPath,
+}: PrepareCacheOptions) {
+  let cachePaths: string[] = [
+    'node_modules/**',
+    'package-lock.json',
+    'yarn.lock',
+  ];
+
+  // add framework specific cache paths
+  const pkg = getPkg(entrypoint, workPath);
+  if (pkg) {
+    const framework = getFramework(pkg);
+
+    if (framework && framework.cachePaths) {
+      cachePaths = [...cachePaths, ...framework.cachePaths];
+    }
+  }
+
+  // glob all cache paths and return
+  let cacheFiles = {};
+
+  for (const cachePath of cachePaths) {
+    cacheFiles = {
+      ...cacheFiles,
+      ...(await glob(cachePath, workPath)),
```

Comment:
This seems inefficient since you are shallow cloning cacheFiles each time. Is there a better way to glob all of these paths?

I think one solution is to change `cachePaths` to be a single string instead of an array of strings.

If a framework has multiple directories that need to be cached, the glob can be written in a way to support that such as `+(.cache/**|static/**|another/**)`.

Does that make sense?

Reply:
Yep! I'll update 🙂